packages/devtools_app/lib/src/config_specific/ide_theme/ide_theme_web.dart

```diff
@@ -27,32 +27,32 @@ IdeTheme getIdeTheme() {
   // If the environment has provided a background color, set it immediately
   // to avoid a white page until the first Flutter frame is rendered.
   if (overrides.backgroundColor != null) {
-    document.body.style.backgroundColor =
-        toCssHexColor(overrides.backgroundColor);
+    document.body!.style.backgroundColor =
+        toCssHexColor(overrides.backgroundColor!);
   }
 
   return overrides;
 }
 
-Color backgroundColor;
-Color foregroundColor;
-double fontSize;
+Color? backgroundColor;
+Color? foregroundColor;
+double? fontSize;
```

Comment:
these don't appear to be used anywhere. let's remove.

Reply:
When I deleted foregroundColor, I got exception in console, when started DevTools in Chrome and connected to app:
https://gist.github.com/polina-c/f5200a8d693619f33b0bb4e44b38803b

Added TODO.